src/libcore/unicode/tables.rs

```diff
@@ -93,7 +93,7 @@ pub mod general_category {
             0x03ff000003ff0000
         ],
         r4: [
-            0, 1, 2, 3, 3, 3, 4, 3, 3, 3, 3, 3, 3, 5, 6, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3,
+            0, 1, 2, 3, 3, 3, 4, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3,
```

Comment:
It’s very worrying that existing tables change without the Unicode version number changing. Is this a bug in `unicode.py`? Or do you get the same changes by re-running unmodified `unicode.py`?

Reply:
Turns out that the files I downloaded before were truncated. Rerunning the version of `unicode.py` in the PR fixed it. Whoops.